pkgs/applications/editors/jetbrains/default.nix

```diff
@@ -183,12 +184,13 @@ rec {
       pname = "rider";
       extraBuildInputs = [
         fontconfig
-        stdenv.cc.cc
-        openssl
         libxcrypt
         lttng-ust_2_12
         musl
+        openssl
+        stdenv.cc.cc
       ];
+      extraLdPath = [ icu openssl ];
```

Comment:
This feels like a regression from https://github.com/NixOS/nixpkgs/pull/257890/commits/bcdfa1884623c42ed83fe541e5a60b8ee652692c if we have to add things to the LD_LIBRARY_PATH again. Typically this suggests that some binaries are not being correctly patched.

I'm a bit confused why the `autoPatchelf` that I added back then is gone completely (even without your changes in this PR)...

Reply:
Thanks for the suggestions, please have another look.

Reply:
This looks better, thanks! You might also want to check and re-apply the follow-up changes that were made:

https://github.com/NixOS/nixpkgs/commit/3b6d08dfdd9a202ff0bc4f2e5051e31d7796025a
https://github.com/NixOS/nixpkgs/commit/812e28f4918122c0d8c349482ee43d73fc0817f4

It's a bit of a mess sadly since I suspect half of the changes are still there but the other half has been lost :S

Reply:
Good catch! I've merged these back in. Happy to help clean it up. :slightly_smiling_face: